src/components/Project/ProjectRow.js

```diff
@@ -13,7 +16,7 @@ export default function ProjectRow(props) {
           type='button'
           className='btn btn-default btn-fab btn-fab-mini btn-round'>
           <i className='material-icons'>edit</i>
-        </button>{' '}
+        </button>
```

Comment:
Here it was on purpose to have distance between the buttons, please do not remove this